src/connection.js

```diff
@@ -1149,6 +1150,55 @@ export class Connection {
     };
   }
 
+  /**
+   * Fetch the contents of a Nonce account from the cluster
+   */
+  async getNonceAndContext(
+    nonceAccount: PublicKey,
+    commitment: ?Commitment,
+  ): Promise<RpcResponseAndContext<NonceAccount>> {
+    const args = this._argsWithCommitment(
+      [nonceAccount.toBase58()],
+      commitment,
+    );
+    const unsafeRes = await this._rpcRequest('getAccountInfo', args);
+    const res = GetAccountInfoAndContextRpcResult(unsafeRes);
+    if (res.error) {
+      throw new Error(res.error.message);
+    }
+    assert(typeof res.result !== 'undefined');
+
+    const isV021 =
+      typeof res.result.context !== 'undefined' &&
+      typeof res.result.value !== 'undefined';
+
+    const slot = isV021 ? res.result.context.slot : NaN;
```

Comment:
I think we can drop v0.21 support anytime now

Reply:
Sounds good. I'll do that in a separate PR.